js/model3D/interactive/Model3DUpload.svelte

```diff
@@ -101,6 +108,19 @@
 			scene,
 			() => {
 				scene.createDefaultCamera(true, true, true);
+				var helperCamera = scene.activeCamera;
```

Comment:
Seems like we need a null check on `helperCamera` for the typescript errors.

Reply:
The current code assumes `scene.createDefaultCamera` succeeds, do you want to add error handling (and in that case, what needs to happen?) or just assume there's an active camera?

I'm pushing a change that keeps assuming the call worked - as a starting point

Reply:
Actually looks like the type error is gone, so should be fine as is.